example/example-permits.ts

```diff
@@ -117,7 +117,7 @@
      console.log('inside: permit_examples');
  
      // Get the contract
-     const permit_address = 'KT18iJy46FCwGDnPNHouVCR5iSacadNniTj8';
+     const permit_address = 'KT1JDbzCiB5yMTMFrWrRvgBfuPRQ6LJ9Ny3A';
```

Comment:
The example fails with :
```
TypeError: contract.methods[entrypoint] is not a function
    at /home/roxaneletourneau/src/ecadlabs/taquito2/taquito/example/example-permits.ts:96:34
    at step (/home/roxaneletourneau/src/ecadlabs/taquito2/taquito/example/example-permits.ts:37:23)
    at Object.next (/home/roxaneletourneau/src/ecadlabs/taquito2/taquito/example/example-permits.ts:18:53)
    at /home/roxaneletourneau/src/ecadlabs/taquito2/taquito/example/example-permits.ts:12:71
    at new Promise (<anonymous>)
    at __awaiter (/home/roxaneletourneau/src/ecadlabs/taquito2/taquito/example/example-permits.ts:8:12)
    at permitParamHash (/home/roxaneletourneau/src/ecadlabs/taquito2/taquito/example/example-permits.ts:123:12)
    at /home/roxaneletourneau/src/ecadlabs/taquito2/taquito/example/example-permits.ts:133:31
    at step (/home/roxaneletourneau/src/ecadlabs/taquito2/taquito/example/example-permits.ts:37:23)
    at Object.next (/home/roxaneletourneau/src/ecadlabs/taquito2/taquito/example/example-permits.ts:18:53)
```

The new contract doesn't have a wrapped entrypoint.

Reply:
I have removed this example from this PR and made a new issue to correctly add the example.